Remove dependency on source-map-support

Using the server SDK in a project produces a warning:
```
WARNING in ./node_modules/source-map-support/source-map-support.js
Module not found: Error: Can't resolve 'module' in '/projects/spark-starter-kit/node_modules/source-map-support'
```

Parsing source maps in the SDK should be a choice by the project devs.

## Context

\-

## Expected Behavior

\-

## Possible Fix

\-

## Your Environment

* Webpack, importing dist/server package.

--- a/load-sourcemaps.js
+++ b/load-sourcemaps.js
@@ -1,1 +1,0 @@
-import 'source-map-support/register'

--- a/webpack.server.js
+++ b/webpack.server.js
@@ -12,11 +12,6 @@ const config = {
     nodeExternals()
   ],
   target: 'node',
-  entry: {
-    index: [
-      resolve(baseDirectoryPath, 'load-sourcemaps.js')
-    ]
-  },
   output: {
     path: serverDistDirectoryPath
   }

